src/applications/facility-locator/components/AppointmentInfo.jsx

```diff
@@ -112,26 +112,30 @@ export default class AppointmentInfo extends Component {
           ),
         );
 
-      return [
-        <li key="specialty-care">Specialty care:</li>,
-        firstThree.map(k =>
-          renderStat(
-            startCase(k.replace(/([A-Z])/g, ' $1')),
-            healthAccessAttrs[k][existing ? 'established' : 'new'],
-            true,
-          ),
-        ),
-        lastToEnd.length > 0 && renderMoreTimes(),
-        <li key="show-more" className="show-more">
-          <button
-            onClick={onClick}
-            className={seeMoreClasses}
-            aria-expanded={this.state[showHideKey] ? 'true' : 'false'}
-          >
-            See {this.state[showHideKey] ? 'less' : 'more'}
-          </button>
-        </li>,
-      ];
+      return (
+        <Fragment>
```

Comment:
Thanks for refactoring this to use a Fragment. `vets-website` should have support for the [short syntax](https://reactjs.org/docs/fragments.html#short-syntax) too.